### DISCUSSION TITLE
iOS Safari – Playback speed of muted videos too fast after 2nd loop

### DISCUSSION THREAD

**Discussion by mlinzner** [vimeo/player.js#1007] [Bug reports]
### Expected Behavior
Muted videos (looping automatically or manual) should have stable playback speed even with repeated playbacks.

### Actual Behavior
On iOS Safari 14.4 and 14.5 beta, muted videos have a strangely fast playback speed after 2nd video loop.
![Actual Behavior Video Example](https://user-images.githubusercontent.com/7240176/108827291-b9bb5080-75c5-11eb-959c-dbfe9aeda473.mov)

### Steps to Reproduce
* Include vimeo video with any configuration causing it to be muted. E.g. muted=1, background=1, …
* Start video and repeat playback 3 times (or wait for the autoloop to reach 3rd loop)

[CodePen Example Editor View](https://codepen.io/mlinzner/pen/LYbONyZ)
[CodePen Example Debug View](https://codepen.io/mlinzner/debug/LYbONyZ)

**Comment by acndr** [comment]
@mlinzner did you manage to resolve this? we are getting the same issue now on 14.7

**Comment by mlinzner** [comment]
@acndr Unfortunately not. We ended up using the default HTML5 video player. In our case the loops do not have visible controls so no one really notices. The only downside is that Vimeo Pro is required for that, else we would be fine with Vimeo Plus.
